1-react/store/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { createStore } from './slomux'
+
+const CHANGE_INTERVAL = 'CHANGE_INTERVAL'
+
+const reducer = (state, action) => {
+  switch(action.type) {
+    case CHANGE_INTERVAL:
+      return state += action.payload
+    default:
+      return {}
+  }
+}
+
+export const changeInterval = value => ({
+  type: CHANGE_INTERVAL,
+  payload: value,
+})
+
+export const store = createStore(reducer)
```

Comment:
Кажется, createStore принимает ещё initialState? Учитывая реализацию редьюсера, предлагаю сделать так:

```suggestion
export const store = createStore(reducer, 0)
```